src/lv_api_map.h

```diff
@@ -236,6 +236,7 @@ static inline void lv_obj_move_background(lv_obj_t * obj)
 #define lv_style_set_transform_zoom         lv_style_set_transform_scale
 
 #define LV_ZOOM_NONE                        LV_SCALE_NONE
+#define lv_coord_t                          int32_t
```

Comment:
It should be handled in line 47 but it seems it was messed up by a search and replace:

```c
typedef int32_t int32_t;
```

should be
```c
typedef int32_t lv_coord_t;
```

Could you fix it there?